modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -1511,6 +1512,18 @@
 		end
 	end
 
+	function m.allModulesPublic(cfg)
+		if _ACTION >= "vs2019" then
+			if cfg.allmodulespublic ~= nil then
+				if cfg.allmodulespublic then
+					m.element("AllProjectBMIsArePublic", nil, "true")
+				else
+					m.element("AllProjectBMIsArePublic", nil, "false")
```

Comment:
Is there an equivalent flag for command line (for premake-core/src/tools/msc.lua)?

Reply:
I don't think so.